app/models/conference.rb

```diff
@@ -15,4 +17,15 @@ def tickets_left
     confirmed_attendances = attendances.select { |attendance| attendance.confirmed }
     tickets.to_i - confirmed_attendances.size
   end
+
+  private
+
+    def date_interval
+      if [starts_on, ends_on].count(&:nil?) == 1
```

Comment:
Hard to say what makes more sense. If you skip the presence validation, you still need to check for the presence of both fields in order to perform the validation on "a before b"...

What's the problem you want to solve after all? Conferences have "invalid" dates, meaning someone basically made a typo when inputting them? Or they don't have dates at all and therefore cannot be processes / used?